composer.json

```diff
@@ -9,7 +9,7 @@
     "guzzlehttp/guzzle": "^6.0",
     "guzzlehttp/psr7": "^1.2",
     "monolog/monolog": "^1.17.1",
-    "phpdocumentor/reflection-docblock": "^3.0.3",
+    "phpdocumentor/reflection-docblock": "^3.0.3|^4.0.0",
```

Comment:
Could you please use `||` like the line below?

Reply:
Hi @ashnewmanjones,

This is still shown as "phpdocumentor/reflection-docblock": "^3.0.3|^4.0.0"

It should be "phpdocumentor/reflection-docblock": "^3.0.3 || ^4.0.0",